src/governor.cairo

```diff
@@ -317,13 +375,18 @@ pub mod Governor {
                 proposal.nay = proposal.nay + weight;
             }
             self.proposals.write(id, proposal);
-            self.vote.write((id, voter), if yea {
-                3
+            
+            // Store vote in appropriate storage for backwards compatibility
+            let vote_value = if yea { 3 } else { 1 };
+            if staker == self.get_staker().contract_address {
+                // For default staker, use old storage for backwards compatibility
+                self.vote.write((id, voter), vote_value);
```

Comment:
we do not need to write to the old storage, only read from old storage, for old votes